scripts/template_upload_all.py

```diff
@@ -390,6 +392,7 @@ def main():
             return 1
         kwargs['stream'] = stream
         kwargs['image_url'] = dir_files[module]
+        kwargs['glance'] = glance
```

Comment:
This is speaking in general, not just to this line.

I'm thinking we should hold off on modifying template_upload_all for the moment and leave it still using the existing ova upload script.

We can run this script manually and do some more testing with sprout before switching over to using glance/qcow2. I think we'd be rushing to modify it in this sprint.